src/components/describe/describeWebview.ts

```diff
@@ -28,11 +56,51 @@ export class DescribePanel extends WebPanel {
     <head>
         <meta charset="UTF-8">
         <title>Kubernetes describe ${this.resource}</title>
+        <script>
+            const vscode = acquireVsCodeApi();
+            const requestRefresh = () => {
+                vscode.postMessage({
+                    command: 'refresh'
+                });
+            };
+
+            window.addEventListener('message', event => {
+                const message = event.data;
+                switch (message.command) {
+                    case 'content':
+                        const elt = document.getElementById('content');
+                        elt.innerText = message.content;
+                }
+            });
+
+            const automaticRefresh = () => {
+                const elt = document.getElementById('refresh');
+                const val = parseInt(elt.value);
+                if (val > 0) {
+                    setTimeout(() => {
+                        requestRefresh();
+                        automaticRefresh();
+                    }, val * 1000);
+                }
+            };
+        </script>
     </head>
     <body>
+        <div style="position: absolute; left: 90%">
```

Comment:
This scrolls off the top if the `describe` output is long enough to merit scrolling.  It should probably be fixed... maybe a toolbar-style design like the one for logs?  (Though that does make the option very in-your-face for what might be quite a niche usage.)

Reply:
removed.